nimbus_nlp/question_classifier.py

```diff
@@ -8,6 +8,9 @@
 import json
 from nimbus_nlp.save_and_load_model import save_model, load_latest_model, PROJECT_DIR
 import json
+from database_wrapper import NimbusMySQLAlchemy
```

Comment:
To avoid double-instantiating the NimbusMySQLAlchemy object, consider removing this and moving get_all_answerable_pairs to the database wrapper.

Reply:
hmm. That's what I've done, `get_all_answerable_pairs` is in the database wrapper. Do i need to instantiate the db differently so I can call it?

Reply:
Sorry, you’re right, ‘get_al_answerable_pairs’ is in the wrapper. The issue is that the database is instantiated at the top level of QA.py, as well as here so there would be two instances running. See the discussion is PR #128 for some future solutions.

If you want a quick and dirty solution you can just do “from QA import db”.